### ISSUE TITLE
npm audit vulnerability

### ISSUE THREAD

**Issue by Jun2theizzle** [young-steveo/bottlejs#116]
When running npm audit on the lastest release, it looks like it has a high vulnerability relating to ` grunt-contrib-jasmine`. Can we expect a fix to this vulnerability soon-ish?

![screen shot 2018-12-07 at 2 29 31 pm](https://user-images.githubusercontent.com/2424525/49671149-8a43bc00-fa2c-11e8-84f9-7121bcbead14.png)

**Comment by young-steveo** [comment]
Yeah, it'll be fixed in the next release.  I wouldn't be too concerned about it though; the vulnerability concerns "Insufficient Entropy" in the crypto lib that the test suite uses for requests locally when running unit tests.  So basically when the test suite runs and requests are made via PhantomJS locally, the "requests" are hackable.  

The bottlejs library itself does not have any dependencies.

**Comment by Jun2theizzle** [comment]
ty
